.golangci.yml

```diff
@@ -118,17 +118,14 @@ linters:
     - unused
     - whitespace
 
-  # don't enable:
-  # - asciicheck
-  # - gochecknoglobals
-  # - gocognit
-  # - godot
-  # - godox
-  # - goerr113
-  # - nestif
-  # - prealloc
-  # - testpackage
-  # - wsl
+  # This list of linters is not a recommendation (same thing for all this configuration file).
+  # We intentionally use limited set of linter:
+  # this configuration file is used with different version of golangci-lint to avoid regressions.
+  # As the linters can change between version,
+  # the configuration may be not compatible or the reports can be different,
+  # and this can break some of our tests.
+  # Also, some linters are not relevant for the project (ex: linters related to SQL).
+  # We have specific constraints, so we use a specific configuration.
```

Comment:
Perhaps we should place this comment at the beginning of the file.

Reply:
Maybe another comment with nearly this comment.
But we need to have a comment on the list of linters.